lib/rules/jsx-curly-spacing.js

```diff
@@ -32,7 +32,52 @@ module.exports = {
     fixable: 'code',
 
     schema: [{
-      enum: SPACING_VALUES
+      definitions: {
```

Comment:
i'm not familiar with the "definitions" rule schema key - what version of eslint added this?